src/um/pdh.rs

```diff
@@ -23,17 +28,166 @@ pub type PDH_HQUERY = HANDLE;
 pub type HQUERY = PDH_HQUERY;
 pub type PDH_HCOUNTER = HANDLE;
 pub type HCOUNTER = PDH_HCOUNTER;
+pub type PPDH_BROWSE_DLG_CONFIG = *mut PDH_BROWSE_DLG_CONFIG;
+pub type PDH_HLOG = HANDLE;
+pub type PPDH_RAW_LOG_RECORD = *mut PDH_RAW_LOG_RECORD;
+pub type PPDH_TIME_INFO = *mut PDH_TIME_INFO;
+pub type PPDH_RAW_COUNTER = *mut PDH_RAW_COUNTER;
+pub type PPDH_COUNTER_INFO_A = *mut PDH_COUNTER_INFO_A;
+pub type PPDH_COUNTER_INFO_W = *mut PDH_COUNTER_INFO_W;
+pub type PPDH_STATISTICS = *mut PDH_STATISTICS;
+pub type PPDH_FMT_COUNTERVALUE_ITEM_A = *mut PDH_FMT_COUNTERVALUE_ITEM_A;
+pub type PPDH_FMT_COUNTERVALUE_ITEM_W = *mut PDH_FMT_COUNTERVALUE_ITEM_W;
+pub type PPDH_DATA_ITEM_PATH_ELEMENTS_A = *mut PDH_DATA_ITEM_PATH_ELEMENTS_A;
+pub type PPDH_DATA_ITEM_PATH_ELEMENTS_W = *mut PDH_DATA_ITEM_PATH_ELEMENTS_W;
+pub type PPDH_FMT_COUNTERVALUE = *mut PDH_FMT_COUNTERVALUE;
+pub type CounterPathCallBack = extern fn(dwArg: DWORD_PTR) -> PDH_STATUS;
 
 STRUCT!{struct PDH_FMT_COUNTERVALUE {
     CStatus: DWORD,
     largeValue: LONGLONG,
 }}
+
+STRUCT!{struct PDH_RAW_LOG_RECORD {
+    dwStructureSize: DWORD,
+    dwRecordType: DWORD,
+    dwItems: DWORD,
+    RawBytes: UCHAR,
+}}
+
+STRUCT!{struct PDH_TIME_INFO {
+    StartTime: LONGLONG,
+    EndTime: LONGLONG,
+    SampleCount: DWORD,
+}}
+
+STRUCT!{struct PDH_RAW_COUNTER {
+    CStatus: DWORD,
+    TimeStamp: FILETIME,
+    FirstValue: LONGLONG,
+    SecondValue: LONGLONG,
+    MultiCount: DWORD,
+}}
+
+STRUCT!{struct PDH_STATISTICS {
+    dwFormat: DWORD,
+    count: DWORD,
+    min: PDH_FMT_COUNTERVALUE,
+    max: PDH_FMT_COUNTERVALUE,
+    mean: PDH_FMT_COUNTERVALUE,
+}}
+
+STRUCT!{struct PDH_FMT_COUNTERVALUE_ITEM_A {
+    szName: LPTSTR,
+    FmtValue: PDH_FMT_COUNTERVALUE,
+}}
+
+STRUCT!{struct PDH_FMT_COUNTERVALUE_ITEM_W {
+    szName: LPWSTR,
+    FmtValue: PDH_FMT_COUNTERVALUE,
+}}
+
+/// For the flags `field`:
+///
+/// bIncludeInstanceIndex    :1
+/// bSingleCounterPerAdd     :1
+/// bSingleCounterPerDialog  :1
+/// bLocalCountersOnly       :1
+/// bWildCardInstances       :1
+/// bHideDetailBox           :1
+/// bInitializePath          :1
+/// bDisableMachineSelection :1
+/// bIncludeCostlyObjects    :1
+/// bShowObjectBrowser       :1
+/// bReserved                :22
+STRUCT!{struct PDH_BROWSE_DLG_CONFIG {
```

Comment:
Should be PDH_BROWSE_DLG_CONFIG_A and also should have PDH_BROWSE_DLG_CONFIG_W added (along with their A/W versions being used in the correct functions.